sqlalchemy_batch_inserts/__init__.py

```diff
@@ -32,16 +32,23 @@ def _has_normal_id_primary_key(base_mapper):
         python_column_type = None
 
     return (
-        primary_key_col.name == "id"
-        and python_column_type == int
-        and primary_key_col.autoincrement in ("auto", True)
-        and primary_key_col.table == base_mapper.local_table
+            python_column_type == int
+            and primary_key_col.autoincrement in ("auto", True)
+            and primary_key_col.table == base_mapper.local_table
```

Comment:
Could you follow the single-indent style from before?

```suggestion
        python_column_type == int
        and primary_key_col.autoincrement in ("auto", True)
        and primary_key_col.table == base_mapper.local_table
```

We should eventually set up a code formatter for this codebase to automate.